### PR TITLE
[onert] Revisits python API 

### PR DESCRIPTION
**Pull request by hseok-oh**
- Rename python package: nnfwapi -> onert
- Native library install path: "nnfwapi/onert" -> "nnfw_onert/natvie"
- Rename binding script: "libnnfw_api_pybind.py" -> "infer.py"
- Rename binding session class: "nnfw_session" -> "session"
- Update documentation
- Update example

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

File: infra/nnfw/python/README.md [hunk 1]
```diff
@@ -39,17 +39,25 @@ $ twine upload --repository-url https://test.pypi.org/legacy/ dist/*
...
 ```
-$ pip install -i https://test.pypi.org/simple/ nnfwapi==0.1.1
+$ pip install -i https://test.pypi.org/simple/ nnfw-onert==0.1.1
 ```
 
 This definition has to be set at the top of the script using nnfw python API.
 
 ```
-from nnfwapi.libnnfw_api_pybind import *
+import nnfw_onert
```
Comment (jiwaszki) [inline infra/nnfw/python/README.md:54]:
```suggestion
from nnfw_onert import onert
```
Wouldn't it be more inline with actual usage? As far as I understand `nnfw_onert` is just a namespace package, so pointing out to shorter path would be even better, just to use `onert.something` later on.

**Review by jiwaszki — COMMENTED** [review]
I really like this idea of renaming! It's making the API so much cleaner from user perspective.

Few small remarks in later comments. More of a clean up than anything.

File: docs/howto/how-to-use-nnfw-python-api.md [hunk 2]
```diff
@@ -17,13 +17,15 @@ Please see [nnfw python api](https://github.com/SAMSUNG/ONE/tree/master/infra/nn
...
+
 # Create session and load nnpackage
 # operations is optional to assign a specific backends to each operation.
 # The default value of backends is "cpu".
 if operations:
-    session = nnfw_session(nnpackage_path, backends, operations)
+    session = nnfw_onert.onert.session(nnpackage_path, backends, operations)
 else:
-    session = nnfw_session(nnpackage_path, backends)
+    session = nnfw_onert.onert.session(nnpackage_path, backends)
```
[the remark this answers is not in the document]
Comment (hseok-oh) [reply docs/howto/how-to-use-nnfw-python-api.md:28]: OK. I'll remove

**Comment by hseok-oh** [comment]
- Remove deprecated code
- Package name: 'nnfw-onert' -> 'onert'
  - Pair with 'onecc'
- Binding script: 'onert.py' -> 'infer.py'
  - We may need train, I/O lib, etc

**Comment by hseok-oh** [comment]
@Samsung/one_onert PTAL

**Review by ragmani — APPROVED** [review]
LGTM :+1:

**Review by jyoungyun — APPROVED** [review]
LGTM

**Pull request merged**